### DISCUSSION TITLE
Splines/Paths/Curves should be converted to mesh objects upon export

### DISCUSSION THREAD

**Discussion by ssnd292** [StjerneIdioten/I3D-Blender-Addon#103] [Ideas & Feature Requests]
Me personally like to keep as many objects as possible easily editable.

That is why I rarely convert splines or paths into actual geometry, since its not needed if they are setup correctly (array and curve modifier for example)

One feature very late down the road I would like to see is the exporter to handle those splines and paths as meshes, so that they are automatically converted into meshes for export.

**Comment by StjerneIdioten** [comment]
GE also has support for some splines/paths so maybe they should be converted into that? Probably depends on the case.
